<launch>
  <node name="rplidarNode"          pkg="rplidar_ros"  type="rplidarNode" output="screen">
  <param name="serial_port"         type="string" value="/dev/rplidar"/>
  <param name="serial_baudrate"     type="int"    value="256000"/><!--A3 -->
  <param name="frame_id"            type="string" value="laser"/>
  <param name="inverted"            type="bool"   value="false"/>
  <param name="angle_compensate"    type="bool"   value="true"/>
  <param name="scan_mode"           type="string" value="Sensitivity"/>
  </node>

  <node pkg="timed_roslaunch" type="timed_roslaunch.sh"
    args="1 project rs_camera.launch"
    name="rs_camera" />

  <node pkg="timed_roslaunch" type="timed_roslaunch.sh"
    args="5 project slam_cartographer.launch"
    name="timed_roslaunch" />
    
  <param name="/offset" type="int" value="-10" />
  <param name="/turning" type="bool" value="False" />

  <param name="/draw_map_o" type="bool" value="True" />
  <param name="/draw_car_o" type="bool" value="True" />
  <param name="/draw_car_way" type="bool" value="True" />
  <param name="/draw_scan_line" type="bool" value="False" />
  <param name="/draw_scan_adge" type="bool" value="False" />
  <param name="/draw_rout" type="bool" value="True" />
  <param name="/img_flip" type="bool" value="True" />

  <node name="slam_control" pkg="project" type="slam_control.py" output="screen" />
  <!--node name="realsense_img" pkg="project" type="realsense_img.py" output="screen" /-->
  <!--node name="auto_motion" pkg="project" type="auto_motion.py" output="screen" /-->
  <!--node name="node_arduino" pkg="project" type="node_arduino.py" output="screen" /-->
  <!--node name="image_stream" pkg="project" type="git_stream.py" output="screen" /-->
  <!--node name="tcp_sender" pkg="project" type="tcp_sender.py" output="screen" /-->

</launch>

